specification/gedcom-3-structures-3-meaning.md

```diff
@@ -953,7 +953,9 @@ See also `INDIVIDUAL_ATTRIBUTE_STRUCTURE`.
 
 #### `NICK` (Nickname) `g7:NICK`
 
-A descriptive or familiar name that is used instead of, or in addition to, one’s proper name.
+A descriptive or familiar name that is used instead of, or in addition to, one’s official or legal name.
+A nickname can be a derivation of another name, such as "Jimmy" from "James", or a distinct name.
+Nicknames can be brief-lived and informal or used consistently even in formal situations throughout one's life.
```

Comment:
The definition text in the spec doesn't use "Nickname", instead using "descriptive or familiar", so this added text may not be appropriate